CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ ExternalProject_Add(tvm-build
 ExternalProject_Add_Step(tvm-build dmlc-build
   DEPENDEES patch
   WORKING_DIRECTORY <SOURCE_DIR>/3rdparty/dmlc-core
-  COMMAND ${CMAKE_COMMAND} --configure -DCMAKE_POSITION_INDEPENDENT_CODE=ON .
+  COMMAND ${CMAKE_COMMAND} -DCMAKE_POSITION_INDEPENDENT_CODE=ON .
```

Comment:
For what did you have to remove the `--configure`?

Reply:
It has caused the following error.
```
CMake Error: Unknown argument --configure
```
My cmake version is
```
❯ cmake --version
cmake version 3.22.1

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```